packages/core/src/Stores/client-store.js

```diff
@@ -333,7 +333,9 @@ export default class ClientStore extends BaseStore {
     @computed
     get is_mt5_allowed() {
         if (!this.landing_companies) return false;
-        return 'mt_financial_company' in this.landing_companies || 'mt_gaming_company' in this.landing_companies;
+        // TODO revert next line when Non-CR MT5 is enabled.
+        // return 'mt_financial_company' in this.landing_companies || 'mt_gaming_company' in this.landing_companies;
+        return 'mt_gaming_company' in this.landing_companies;
```

Comment:
why are we making this change? do you mean non-cr accounts all don't have mt_gaming_company?

Reply:
Yes, is this assumption incorrect?

Reply:
we shouldn't rely on that. we should check that client has svg landing company or if is virtual then their gaming or financial company is svg

Reply:
Got you, I'm on it.